### PR TITLE
bluetooth: controller: add support for HCIVSBigReservedTimeSet

### PR DESCRIPTION
**Pull request by adamfc2000**
Add support for a VS HCI command to adjust the reserved time in a BIG.

### CODE REVIEW

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst [hunk 1]
```diff
@@ -24,6 +24,7 @@ Changes
 * Fixed an issue where the peripheral did not always accept timing values from the same central.
 * Fixed an issue where the PA sync skip parameter was not used.
 * Fixed an issue where it was not possible to create BIG sync after terminating a pending BIG sync.
+* Added support for HCIVSBigReservedTimeSet command and corresponding Kconfig option BT_CTLR_BIG_RESERVED_TIME_US.
```
Comment (sean-madigan) [inline doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:27]: Don't think this is the correct place for this

File: subsys/bluetooth/controller/Kconfig [hunk 2]
```diff
@@ -508,5 +508,13 @@ config BT_CTLR_SDC_IGNORE_HCI_ISO_DATA_TS_FROM_HOST
 	  not add timestamps to the HCI ISO data packets if the timestamp is not based on the clock
 	  of the controller.
 
+config BT_CTLR_BIG_RESERVED_TIME_US
+	int "BIG reserved time [us]"
+	range 0 4000000
+	default 2500
+	help
+		The default time reserved at the end of a BIG event for periodic advertising
```
Comment (sean-madigan) [inline subsys/bluetooth/controller/Kconfig:516]: Formatting seems to be off

File: doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst (hunk 1 above)
Comment (sean-madigan) [reply doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst:27]: I think we'll have to merge it to internal repo changelog

**Review by sean-madigan — CHANGES_REQUESTED** [review]

File: subsys/bluetooth/controller/Kconfig [hunk 3]
```diff
@@ -508,5 +508,13 @@ config BT_CTLR_SDC_IGNORE_HCI_ISO_DATA_TS_FROM_HOST
 	  not add timestamps to the HCI ISO data packets if the timestamp is not based on the clock
 	  of the controller.
 
+config BT_CTLR_BIG_RESERVED_TIME_US
```
Comment (sean-madigan) [inline subsys/bluetooth/controller/Kconfig:511]:
```suggestion
config BT_CTLR_SDC_BIG_RESERVED_TIME_US
```

File: subsys/bluetooth/controller/Kconfig [hunk 4]
```diff
@@ -534,5 +534,13 @@ config BT_CTLR_SDC_IGNORE_HCI_ISO_DATA_TS_FROM_HOST
 	  not add timestamps to the HCI ISO data packets if the timestamp is not based on the clock
 	  of the controller.
 
+config BT_CTLR_SDC_BIG_RESERVED_TIME_US
+	int "BIG reserved time [us]"
+	range 0 4000000
+	default 2500
+	help
+	  The default time reserved at the end of a BIG event for periodic advertising
```
Comment (sean-madigan) [inline subsys/bluetooth/controller/Kconfig:542]: Maybe update wording to match Rubin's suggestions in bitbucket PR

**Review by sean-madigan — APPROVED** [review]

File: subsys/bluetooth/controller/Kconfig [hunk 5]
```diff
@@ -534,5 +534,12 @@ config BT_CTLR_SDC_IGNORE_HCI_ISO_DATA_TS_FROM_HOST
 	  not add timestamps to the HCI ISO data packets if the timestamp is not based on the clock
 	  of the controller.
 
+config BT_CTLR_SDC_BIG_RESERVED_TIME_US
+	int "BIG reserved time [us]"
+	range 0 4000000
+	default 2500
```
Comment (sean-madigan) [inline subsys/bluetooth/controller/Kconfig:540]: Maybe it makes sense to default this to the periodic adv event length?

**Review by rugeGerritsen — APPROVED** [review]

**Pull request merged**